src/modules/expense/Form.tsx

```diff
@@ -0,0 +1,524 @@
+              ? 'Repeat transaction, set your own time'
+              : 'Repeat transaction'}
+          </Typography>
+        </StyledDivRepeatSection>
+        <Switch checked={switchChecked} onChange={handleSwitchChecked} />
+      </StyledDiv>
+      {showContinueButton && (
+        <Button
+          onClick={handleContinueButton}
+          variant='contained'
+          sx={{
+            padding: theme.spacing(2),
+            textTransform: 'none',
+            fontSize: '16px',
+          }}
+        >
+          Continue
+        </Button>
+      )}
+      {showAttachmentComponent && (
+        <StyledAttachmentArea
+          direction='row'
+          spacing={2}
+          padding={3}
+          justifyContent='space-evenly'
+        >
+          <AttachmentOptionsButtons
+            icon={<CameraAltRoundedIcon sx={{ color: colors.violet[100] }} />}
+            text='Camera'
+            onClickHandler={() => {
+              setShowContinueButton(!showContinueButton);
+              setShowAttachmentComponent(false);
+            }}
+          />
+          <AttachmentOptionsButtons
+            icon={<ImageRoundedIcon sx={{ color: colors.violet[100] }} />}
+            text='Image'
+            onClickHandler={() => {
+              setShowContinueButton(!showContinueButton);
+              setShowAttachmentComponent(false);
+            }}
+          />
+          <AttachmentOptionsButtons
+            icon={
+              <InsertDriveFileRoundedIcon sx={{ color: colors.violet[100] }} />
+            }
+            text='Document'
+            onClickHandler={() => {
+              setShowContinueButton(!showContinueButton);
+              setShowAttachmentComponent(false);
+            }}
+          />
+        </StyledAttachmentArea>
+      )}
+      {toggleContinueButton && !switchChecked && (
+        <Modal
+          open={showPopup}
+          onClose={handlePopupClose}
+          aria-labelledby='modal-modal-title'
+          aria-describedby='modal-modal-description'
+        >
+          <Box sx={popupStyle}>
+            <CheckCircleIcon
+              fontSize='large'
+              sx={{ margin: 'auto', color: colors.violet[80] }}
+            />
+            <Typography id='modal-modal-description' sx={{ mt: 2 }}>
+              Transaction has been succesfully added
+            </Typography>
+          </Box>
+        </Modal>
+      )}
+      {toggleContinueButton && switchChecked && (
+        <StyledRepeatOptionsSection>
+          <FormControl>
+            <InputLabel id='select-frequency'>Frequency</InputLabel>
+            <Select
+              labelId='select-frequency'
+              id='simple-select-frequency'
+              value={selectedFrequency}
+              onChange={handleChangeFrequency}
+              input={
+                <OutlinedInput id='select-single-frequency' label='frequency' />
+              }
+            >
+              <MenuItem value='Daily'>Daily</MenuItem>
```

Comment:
Don't repeat elements like this. Instead, save options in an array (`const frequencyOptions = ['Daily', 'Weekly'....];`) then map through the array.